components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2396,6 +2397,21 @@ void RewardsServiceImpl::ShowNotificationAddFunds(bool sufficient) {
       "rewards_notification_insufficient_funds");
 }
 
+void RewardsServiceImpl::MaybeShowNotificationTipsPaid() {
+  GetAutoContribute(base::Bind(&RewardsServiceImpl::ShowNotificationTipsPaid,
```

Comment:
this should be bind once

Reply:
@NejcZdovc you just used `base::Bind` here for the exact same function, is that wrong?

Reply:
yup, we should use once when we only want one response

Reply:
done, now using BindOnce :)